static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -17,7 +17,7 @@ import InviteStatusMessage from 'sentry/components/modals/inviteMembersModal/inv
 import {ORG_ROLES} from 'sentry/constants';
 import {IconAdd} from 'sentry/icons';
 import {t} from 'sentry/locale';
-import SentryTypes from 'sentry/sentryTypes';
+import * as SentryPropTypes from 'sentry/sentryPropTypeValidators';
```

Comment:
I see the PR does named `import {isObject}`, so didn't read the whole PR and went top to bottom updating everything to be named instead of using the `*` import. Then i realized that more files in the PR started with `*`, it would've been easier to go the other way.

Up to you to accept or reject the suggestions , i don't mind.

```suggestion
import {isOrganization} as SentryPropTypes from 'sentry/sentryPropTypeValidators';
```

Reply:
Yeah, I'm going to drop this and make it confirm to the old signature so we'll just import {SentryPropTypes} and access via SentryPropTypes.Organization etc. I don't want to pollute isObject or isArray namespaces as they are also helpers from lodash.

Reply:
we could also use move custom/unique names. `isObjectProps`

no one should be googling or expecting that this is the real PropTypes library (in case they expect more things are available for example). 

I'm looking at it with the perspective that we're not going to be adding more PropType callsites, but it'll take a while to replace these components with functional ones.

Reply:
Yeah, I'm hoping that as we export under a single object, you'll always need to import {SentryPropTypes} and so it wont conflict with anything. I suspect that writing custom validators will deter people from using this 😅

Reply:
yeah, i fully don't expect anyone to write their own validator

![4818000](https://github.com/getsentry/sentry/assets/187460/1182fc64-677f-44b3-9770-89f0648b4c7e)